DEPLOYMENT.md

```diff
@@ -0,0 +1,80 @@
+# Airtable Gateway deployment guide
+
+
+We at MANYC are presently only supporting [paid Heroku dynos](https://www.heroku.com/pricing).
+This project is an 'active' server that actively polls your airtable every 10 seconds. Unfortunately  the free Heroku tier won't let you poll for any period of time before the free Heroku dyno goes to [sleep](https://devcenter.heroku.com/articles/free-dyno-hours#dyno-sleeping)
+
+We hope in the future to have FULL deployment instructions on alternative platforms.
```

Comment:
Like this section, we can def have separate sections for each type of deployment